src/Explain.h

```diff
@@ -85,7 +86,7 @@ class ScreenBuffer {
     void print(std::ostream& os) {
         if (height > 0 && width > 0) {
```

Comment:
For marginally clearer purpose and less nesting, this test is better as one that returns immediately if height/width aren't valid.
```
if (height <= 0 || width <= 0) {
    return;
}
```